app/Http/Requests/StoreUser.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StoreUser extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize(): bool
+    {
+        return true;
```

Comment:
You could extend my BaseRequest class to your Requests once you'll rebase the current dev.
That way you won't need to copy authorize() method (we're using middleware for that) and you'll get automatic HTTP 422 with error messages if the validation failed.